cmd/aries-agent-mobile/pkg/wrappers/aries.go

```diff
@@ -0,0 +1,66 @@
+/*
+Copyright SecureKey Technologies Inc. All Rights Reserved.
+
+SPDX-License-Identifier: Apache-2.0
+*/
+
+package wrappers
+
+import (
+	"fmt"
+
+	commandintroduce "github.com/hyperledger/aries-framework-go/pkg/controller/command/introduce"
+	"github.com/hyperledger/aries-framework-go/pkg/didcomm/transport/ws"
+	"github.com/hyperledger/aries-framework-go/pkg/framework/aries"
+)
+
+// Options for initializing an Aries framework instance
+//type Options struct {
+//
+//}
+
+// Aries ...
+type Aries struct {
+	framework *aries.Aries
+
+	Introduce *Introduce
+}
+
+// NewAries ...
+func NewAries() (*Aries, error) {
+	inbound, err := ws.NewInbound("localhost:5050", "ws://example.com:5050")
+	if err != nil {
+		return nil, fmt.Errorf("failed to create websocket: %w", err)
+	}
+
+	// TODO properly receive options
+	// how does this work for REST, especially since a StoreProvider is required?
+	// also are these options static and can they be changed during runtime?
+	framework, err := aries.New(aries.WithInboundTransport(inbound))
+	if err != nil {
+		return nil, fmt.Errorf("failed to initialize Aries framework: %v", err)
+	}
+	return &Aries{framework: framework}, nil
+}
+
+// LoadIntroduce ...
+func (a *Aries) LoadIntroduce() error {
+	context, err := a.framework.Context()
+	if err != nil {
+		return fmt.Errorf("failed to get Framework context: %v", err)
+	}
+
+	command, err := commandintroduce.New(context)
+	if err != nil {
+		return fmt.Errorf("failed to create introduce command instance: %v", err)
+	}
+
+	a.Introduce.command = command
+
+	return nil
+}
```

Comment:
I think we do not need to load each client. Try to use `GetCommandHandlers` instead. it will provide all everything.